### PR TITLE
Wires up pusher to deal with channel updates

### PR DESCRIPTION
**Pull request by dzucconi**
I'm going to use this packaged version of the Pusher lib in lieu of the CDN one as well. I'll remove that before merging.

pushed

### CODE REVIEW

**Comment by broskoski** [comment]
🔥

File: react/components/ChannelContents/index.js [hunk 1]
```diff
@@ -54,16 +52,14 @@ export default class ChannelContents extends PureComponent {
...
-                skeleton={newBlocks}
-              />
-            }
           </React.Fragment>
         }
 
+        <ChannelContentsAppended
+          id={id}
+          pendingSkeleton={pendingSkeleton}
+        />
```
Comment (dzucconi) [inline react/components/ChannelContents/index.js]: This winds up making way more sense: we don't want to rechunk the skeleton and we'd get the exact same result if we did.

Comment (broskoski) [reply react/components/ChannelContents/index.js]: Nice, yes.

File: react/components/ChannelContents/components/ChannelContentsAppended/index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import { withApollo } from 'react-apollo';
+import sharify from 'sharify';
+
+import initPusherClient from 'react/util/initPusherClient';
+
+import channelContentsSetQuery from 'react/components/ChannelContents/components/ChannelContentsSet/queries/channelContentsSet';
+
+import ChannelContentsSet from 'react/components/ChannelContents/components/ChannelContentsSet';
+
+const { data: { NODE_ENV } } = sharify;
```
Comment (dzucconi) [inline react/components/ChannelContents/components/ChannelContentsAppended/index.js]: If testing this locally you have to be mindful of the environment you're running in vs the server's.

Comment (dzucconi) [reply react/components/ChannelContents/components/ChannelContentsAppended/index.js]: By mindful I mean you won't be subscribed to the correct channels.

**Pull request merged**